README.md

```diff
@@ -1,194 +1,111 @@
-
-``` bash
-oc edit dsc
-      trainingoperator:
-        managementState: Managed
-```
-
-To verify the PytorchJob is available in the cluster run the following.
+This project makes running the InstructLab large language model (LLM) fine-tuning process easy and flexible on OpenShift. However, before getting started there are a few prerequisites and additional setup steps that need to be completed.
+
+## Requirements
+
+* An OpenShift cluster with:
+  * Sufficient GPUs available for training.
+    * At a minimum, a node with at least 4 GPUs such as an NVIDIA A100s
+    * This is not including GPUs required to deploy and run Judge & Teacher models (see below)
+  * The following Operators already installed:
+    * Red Hat Authorino
+    * Red Hat Openshift Serverless
+    * Red Hat OpenShift Service Mesh v2
+      * NOTE: v3 is not compatible with RHOAI
+* [SDG taxonomy tree] to utilize for Synthetic Data Generation (SDG)
+* An OpenShift AI installation, with:
+  * Training Operator, ModelRegistry, KServe, and Data Science Pipelines components installed via the DataScienceCluster
+    * See docs on [Installing RHOAI components via DSC]
+  * For [Model Registry] you will need:
+    * Model Registry API URL
+    * Model Registry Name
+  * A data science project/namespace, in this document this will be referred to as `<data-science-project-name/namespace>`
+    * The Data Science Project should have a [Data Science Pipelines Server Configured]
+  * A GPU [Accelerator profile enabled and created]
+    * NOTE: Install NVIDIA GPU Operator 24.6
+        * This is due to a [known issue with CUDA Driver] versions mismatch, you can only use this Operator version.
+* A [StorageClass] that supports dynamic provisioning with [ReadWriteMany] access mode.
+  * You can deploy your own using [nfs storage] for non production use cases
+* Have the location for the [base model configured] (i.e. S3 or OCI)
+* A locally installed [oc] command line tool to create and manage kubernetes resources.
+* [Teacher and Judge models] being served with their access credentials stored as k8s secrets in `<data-science-project-name/namespace>`
+* An OCI registry to push the output model to along with credentials with push access
+
+#### Disconnected Cluster Requirements
+
+Follow the [disconnected setup instructions] for setting up your disconnected environment.
+
+## Enable the pipeline
+
+The InstructLab pipeline is automatically provisioned and managed by the DataSciencePipelinesApplication (DSPA) resource. This management is disabled by default, you may enable it by patching the following field in the DSPA:
 
 ```bash
-oc get pytorchjob
+DS_PROJECT="<data-science-project-name/namespace>"
+DSPA_NAME="dspa" # RHOAI Default name for DSPA
+oc patch dspa ${DSPA_NAME} -n ${DS_PROJECT} --type=merge --patch='{"spec": { "apiServer": { "managedPipelines": { "instructLab": {"state": "Managed"}}}}}'
```

Comment:
```suggestion
oc patch dspa ${DSPA_NAME} -n ${DS_PROJECT} --type=merge --patch='{"spec": { "apiServer": { "managedPipelines": { "instructLab": {"state": "Managed"}}}}}'
After a few seconds, the InstructLab pipeline will be automatically added to the pipeline server and it will be available in RHOAI Dashboard.
```